Source/FieldSolver/SpectralSolver/SpectralAlgorithms/PsatdAlgorithmPml.H

```diff
@@ -21,27 +21,40 @@
 
 #if WARPX_USE_PSATD
 
-/* \brief Class that updates the field in spectral space
+/*
+ * \brief Class that updates the field in spectral space
  * and stores the coefficients of the corresponding update equation.
  */
 class PsatdAlgorithmPml : public SpectralBaseAlgorithm
 {
     public:
-        PsatdAlgorithmPml(const SpectralKSpace& spectral_kspace,
-                          const amrex::DistributionMapping& dm,
-                          const SpectralFieldIndex& spectral_index,
-                          int norder_x, int norder_y,
-                          int norder_z, short grid_type,
-                          amrex::Real dt,
-                          bool dive_cleaning,
-                          bool divb_cleaning);
 
-        void InitializeSpectralCoefficients(
+        /*
+         * TODO Add Doxygen
```

Comment:
Could you add the docstrings?

Reply:
Yes, thanks, done in [afb8c86](https://github.com/ECP-WarpX/WarpX/pull/733/commits/afb8c869348ebd0fb6ff64dfdc594008e8ea0798).